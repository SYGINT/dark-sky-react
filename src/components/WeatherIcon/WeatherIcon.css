.weather-icon {
  width: 50px !important;
  height: 50px !important;
  background-position: center center;
  background-size: 200%;
}
