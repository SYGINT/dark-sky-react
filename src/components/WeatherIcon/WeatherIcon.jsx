import React from "react";

import "./WeatherIcon.css";

function WeatherIcon(props) {
  return (
    <div
      className="avatar weather-icon"
      style={{
        backgroundImage: `url('/images/clear-day.svg')`
      }}
    />
  );
}

export default WeatherIcon;
