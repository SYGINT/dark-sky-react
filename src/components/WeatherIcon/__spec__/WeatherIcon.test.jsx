import React from "react";
import { shallow } from "enzyme";

import WeatherIcon from "../";

it("renders without crashing", () => {
  const wrapper = shallow(<WeatherIcon icon="clear-day" />);
  expect(wrapper).toMatchSnapshot();
});
