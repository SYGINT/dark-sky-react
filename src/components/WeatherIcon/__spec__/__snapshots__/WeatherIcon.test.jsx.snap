// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="avatar weather-icon"
  style={
    Object {
      "backgroundImage": "url('/images/clear-day.svg')",
    }
  }
/>
`;
